packages/node/README.md

```diff
@@ -20,8 +20,8 @@ Just add the middleware to [Express](https://expressjs.com/), and that's it!
 ```javascript
 const readme = require('readmeio');
 
-app.use(readme.metrics('<<apiKey>>', (req, res) => ({
-  id: req.<userId>,
+app.use(readme.metrics(README_API_KEY, (req, res) => ({
```

Comment:
This _might_ (I can't remember if we set this up or not) get pulled in automatically to the documentation, in which case the <<apiKey>> syntax will be replaced with the users actual ReadMe API Key!

Not sure if we are actually syncing this stuff though or if that was just an idea we had

Reply:
oh that's a good point, I believe you're right!

Reply:
@mjcuva is there a reason we don't do this in the other README's? Sounds like we probably should...

Reply:
Yeah agreed! No clue why we aren't doing it other places, it might just be because the node one was written by someone at ReadMe whereas python/ruby were written by contractors.

Reply:
cool, I can update the others then